Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -216,7 +217,8 @@ public function build(array $buildSubject): array
             unset($requestBody['installments']);
         }
 
-        $requestBody['additionalData']['allow3DS2'] = true;
+        $requestBody['additionalData']['allow3DS2'] =
```

Comment:
I see that the field in deprecated in the API, is this a concern? https://docs.adyen.com/api-explorer/Checkout/71/post/payments#request-additionalData-AdditionalData3DSecure-allow3DS2

Reply:
Hey @acampos1916, thank you for the review.

This is not a concern, yet. Though, we have a separate ticket to handle it. The new field is currently wired to the legacy one in Checkout API backend. So, we are still safe.